base/logger_file.go

```diff
@@ -138,7 +141,7 @@ func (lfc *FileLoggerConfig) init(level LogLevel, logFilePath string, minAge int
 
 	if lfc.Output == nil {
 		lfc.Output = newLumberjackOutput(
```

Comment:
Are we (and support) OK with compressing (gzip) rotated stats files? If not, we'll need to specify Compress: false inside `newLumberjackOutput` for the stats logger.

Reply:
Forget about this one, @adamcfraser reminded me that `sgcollect_info` will extract it before support tooling even sees it.

Reply:
To understand this better, what would be the issue w/ compressing rotated stats files?  I'd expect Mortimer or other tooling to decompress as needed.